app/views/polls/index.html.haml

```diff
@@ -2,15 +2,25 @@
 %table
   %tbody
     %tr
-      Open Polls:
+      Date Started:
+    %tr
+      Currently active:
+    %tr
+      Winner:
     -@polls.each do |poll|
       %tr
         %td
           #{ poll.created_at }
         %td
+          #{ !poll.closed }
+        %td
+          #{ poll.winner.try(:brand) }
```

Comment:
You may not need the `try` here. Earlier you validate that beer brands have at least 3 letters, which is also like validating that they are present

In either case we always use the `try!` which will throw an error if the method you are trying doesnt actually exist 
```
Beer.try(:shenanigans) # => nil
Beer.try!(:hocus_pocus) # => Raises and error, Beer does not implement the hocus_pocus method
```